src/thermo/Nasa9PolyMultiTempRegion.cpp

```diff
@@ -57,6 +57,43 @@ Nasa9PolyMultiTempRegion::Nasa9PolyMultiTempRegion(vector<Nasa9Poly1*>& regionPt
     }
 }
 
+Nasa9PolyMultiTempRegion::Nasa9PolyMultiTempRegion(double tlow, double thigh, double pref,
+                                                   const double* coeffs)
+{
+    size_t regions = (size_t)coeffs[0];
```

Comment:
We usually try (admittedly not consistent, especially in older code) to use `static_cast` rather than the C-style casts, i.e.
```suggestion
    size_t regions = static_cast<size_t>(coeffs[0]);
```